src/main/java/com/snowflake/kafka/connector/records/SnowflakeAvroConverter.java

```diff
@@ -69,6 +75,32 @@ void readBreakOnSchemaRegistryError(final Map<String, ?> configs) {
     }
   }
 
+  void readReaderSchema(final Map<String, ?> configs)
```

Comment:
Can we add comment here? (I know it is self explanatory but going through docstrings is much faster than through code.

Reply:
Sure!